CliClient/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "joplin",
-  "version": "1.0.149",
+  "version": "1.0.150",
```

Comment:
I'm not really a Node expert, is this version bump expected when upgrading the node version in `package.json`? If not, I can remove this change.

Reply:
Yep, this (the version bump) is done during the release process. Please revert this change.